tasks/configure.yml

```diff
@@ -9,6 +9,16 @@
     mode: 0700
   register: pgdata_dir_exist
 
+- name: PostgreSQL | Get locales
+  shell: locale -a
+  register: locales
+  changed_when: false
+
+- name: PostgreSQL | Ensure the locale is generated
+  sudo: yes
+  shell: locale-gen {{postgresql_locale.split('.').0}}
+  when: "'{{postgresql_locale.split('.').0 }}' not in locales.stdout"
+
```

Comment:
Cool, will do this tonight and convert this to use the builtin for local generation, thanks for the heads up @sebalix